package.json

```diff
@@ -93,40 +93,43 @@
       "command": "web-dev-server --open --watch",
       "dependencies": [
         "patch",
-        "compile",
         "analyze",
         "playgrounds",
-        "watch:docs"
+        "11ty-dev"
+      ]
+    },
+    "11ty-dev": {
+      "service": true,
+      "command": "NODE_OPTIONS='--import tsx/esm' eleventy --serve --incremental --config=eleventy.config.ts",
+      "dependencies": [
+        "analyze"
       ]
     },
     "dev": {
       "service": true,
       "command": "web-dev-server --watch",
       "dependencies": [
         "patch",
-        "compile",
         "analyze"
       ]
     },
     "serve": {
       "service": true,
-      "command": "NODE_OPTIONS='--import tsx/esm' eleventy --config=eleventy.config.ts --serve --incremental",
+      "command": "TSX_TSCONFIG_PATH=tsconfig.settings.json NODE_OPTIONS='--import tsx/esm' eleventy --config=eleventy.config.ts --serve --incremental",
       "dependencies": [
         "patch",
-        "compile",
         "analyze",
         "playgrounds"
       ]
     },
     "build": {
       "dependencies": [
         "patch",
-        "compile",
         "analyze",
         "react-wrappers",
         "entrypoint",
         "copy-assets",
-        "docs"
+        "compile"
```

Comment:
Unless I'm wrong we'd still need `npm run build` to build the docs in CI unless you are planning that change an haven't made it yet.

Reply:
Why? The while point of this PR is that 11ty shouldn't require a build

Reply:
Ok, well it's working in the DP but I think @adamjohnson and I both are confused then with the files locally and how they get generated.  Maybe a walk through of how things are working is necessary here.

Reply:
in serve mode, passthroughCopy works without writing the files to disk

Reply:
Ok well that explains the unfamiliar behavior.  I expected to see those files and we were definitely confused.  So I think I need to understand the mode switching and what to expect.